_posts/op-ed/2022-11-01-What-is-High-Priority.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: What is High Priority?
+description: >
+  "That doesn't matter!" – it's said far too often, isn't it? But did you know that we often don't consider facts and conditions when we think about priorities?
+date: 2022-11-01 16:00:00 +0100
+image: '/images/posts/prism-crystal.jpg'
+tags: [op-ed, time, management, organization]
+featured: false
+---
+
+### The <font color="#F88">Time Management</font> Series
+
+This post is just one part of my [Time Management](/2022/11/01/Time-is-of-the-Essence) series. I really recommend you read the whole series if you have time, but I'll do my best to make this part self-contained.  
+**Important disclaimers** are also included there.
+
+-----
+
+## A bullet-proof framework
+
+#### 🎯 &nbsp; About goals…
+
+More controversial opinions: ✅
+
+There are a few things I consider when setting goals for myself and my organization[s]:
+
+1. The _more responsibility_ I bear for my decisions, the _less_ often I have to make decisions. Make sure those choices are good 
+2. I can do things right in many different ways, often _without_ going through the _known/proven steps_. Skip steps when possible 
+3. I've become _brutally honest_ with myself about my capacity and time commitment. This is to the point that I _give up_ on most (90%) of my ideas and external requests. This is good, be transparent and reject things
+
+**What do you disagree with the most?!**
+
+Either way, I think I can justify most of the things I just wrote, even though my framework isn't great. 😄 But it's a good starting point if we're looking for a reliable algorithm to prioritize tasks, I think.
+
+So what are we looking for?
+
+```kotlin
+fun findPriorityItem(items: List<WorkItem>): WorkItem {
+  // FIXME: This is what we're looking for
+  error("Too hard to implement")
+}
+```
+
+#### 🌱 &nbsp; The basic framework
+
+Visualizations are nice, but let's also clarify. Remember that all classifications here are **relative** and not absolute, we're always comparing work items to other work items on the table.
+
+{:refdef: style="text-align: center;"}
+![Priority Matrix](/images/posts/time-management-priority-table.png)
+*The Priority Matrix (Nike don't sue me)*
+{: refdef}
+
+**1. Urgent, but _Not_ Impactful**: "Just do it."  
+These are the things I really don't want to do, because I don't think they're important (yet). But these things tend to escalate quickly if ignored. For example, my day-to-day work includes updating security libraries, understanding a rare bug in the Canary deployment, investigating a sudden drop in analytics events or a spike in costs, approving access requests, etc. I usually force myself to do these things right away because they get done quickly and don't mess up my schedule at all. Waiting can cause them to be moved to the next category here, which I don't want.
```

Comment:
Clarify what you mean by they don't take much time - is there a category of tasks that is high urgency, low priority but takes a lot of time? What should be done about them?